Gems/Multiplayer/Code/CMakeLists.txt

```diff
@@ -79,7 +79,7 @@ ly_add_target(
 
 # The  "Multiplayer" target is used by clients and servers, Debug is used only on clients.
 ly_create_alias(NAME Multiplayer.Clients NAMESPACE Gem TARGETS Gem::Multiplayer Gem::Multiplayer.Debug)
-ly_create_alias(NAME Multiplayer.Servers NAMESPACE Gem TARGETS Gem::Multiplayer)
+ly_create_alias(NAME Multiplayer.Servers NAMESPACE Gem TARGETS Gem::Multiplayer Gem::Multiplayer.Debug)
```

Comment:
I keep doing this locally as well, we should probably just do this until the server target actually becomes headless.